### PR TITLE
[INTERNAL] Update slack badge

### PR DESCRIPTION
**Pull request by RandomByte**
The link changed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ![UI5 logo](./docs/images/UI5_logo_wide.png)
 
 # [UI5](https://ui5.sap.com) Build and Development Tooling
-[![OpenUI5 Community Slack (#tooling channel)](https://slackui5invite.herokuapp.com/badge.svg)](https://slackui5invite.herokuapp.com)
+[![OpenUI5 Community Slack (#tooling channel)](https://img.shields.io/badge/slack-join-yellow.svg)](https://join-ui5-slack.herokuapp.com)
```
Comment (matz3) [inline README.md]: Why yellow? 😄

Comment (RandomByte) [reply README.md]: Couldn't find a green that matches the one of the travis badge :expressionless:

Comment (matz3) [reply README.md]: What about this one? (Same color as the previous badge)

`https://img.shields.io/badge/slack-join-e01563.svg`

Comment (RandomByte) [reply README.md]: Let's actually match the travis badge green 👍

**Pull request merged**